src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 24px;
+
+  &__rating-box {
+    display: flex;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__reviews {
+    margin: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: $mainTextColor;
+    font-style: $fontStyle;
+    display: flex;
+    align-items: center;
+  }
+
+  @for $active from $starStart through $starEnd {
+    &__star--star-#{$active - $substrictionAmount} {
```

Comment:
you need to use nth-child instead of this loop

Reply:
pseudo-class nth-child added